panda/src/dxgsg9/dxTextureContext9.cxx

```diff
@@ -1935,7 +1935,7 @@ fill_d3d_texture_pixels(DXScreenData &scrn, bool compress_texture) {
                 scaled *= 255;
                 color = D3DCOLOR_RGBA((int)scaled[0], (int)scaled[1], (int)scaled[2], (int)scaled[3]);
                 flags = D3DCLEAR_TARGET;
-                if (device -> Clear (NULL, NULL, flags, color, 0.0f, 0) == D3D_OK) {
+                if (device -> Clear (nullptr, nullptr, flags, color, 0.0f, 0) == D3D_OK) {
```

Comment:
First argument is a `DWORD`, so should be `0`, or it does not compile.

https://msdn.microsoft.com/en-us/library/windows/desktop/bb174352.aspx